README.md

```diff
@@ -84,42 +100,57 @@ export default class StepPlanet extends Steps {
 }
 ```
 
-The inheritance of the `Steps` class expose following class fields:
+The manager exposed his options to each steps, they can access it with `this.options`.
+The inheritance of the `Steps` class exposes following the class fields:
+
+#### `id`
+
+`String`
+
+The route identifier, an alias to used inside the app instead of the route.
+
+_The parameter is a public instance fields._
 
 #### `route`
 
 `String`
 
-Route for step navigation. `StepManager` use hash for steps navigation.
+The route for the step navigation. `StepManager` use hash for steps navigation (`#people`).
```

Comment:
_`StepManager` uses hashes for steps navigation_

Reply:
C'est corrigé